src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -63,6 +66,18 @@ public void update(StandaloneConfig standaloneConfig, List<String> ignoreGlobLis
         setAuthorDisplayNameMap(newAuthorDisplayNameMap);
     }
 
+    /**
+     * Checks for duplicate aliases in {@code authorEmailsAndAliasesMap} and generates warnings
+     * @param authorEmailsAndAliasesMap
+     * @param alias
+     */
+    public void checkDuplicateAliases(Map<String, Author> authorEmailsAndAliasesMap, String alias) {
+        if (authorEmailsAndAliasesMap.containsKey(alias)) {
+            logger.warning(String.format(
+                    "Duplicate alias %s found. The alias will belong to the last author who claims it", alias));
```

Comment:
I think can provide the last author's git id in the log as well, just to let the user know the specific author that gets the alias

Reply:
I might have to send one extra variable through all functions for this. Works? 
Not sure if I have the information currently to display the gitId without the extra variable

Reply:
yes or you can just add the `if` check within the `update` method, instead of creating another method. Don't think it adds much value to separate little logic to another method.

Reply:
Actually I am calling the function from two places, therefore, extracted out to a function. One for csv, one for config.json

Reply:
ok, then you'll have to pass the `author id` to the checkDuplicateAliases method